### PR TITLE
Xrt kvs mrtk

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
# XRTK - Mixed Reality Toolkit Change Request

## Overview

New MRTK vs XRTK guidance.

### CODE REVIEW

**Comment by SimonDarksideJ** [comment]
Will move and update this once the getting started docs are in to avoid conflicts

File: articles/A02-ChoosingXRTKvsMRTK.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Core Differences
+
+Where the two frameworks truly diverge, is an adherence to the core architecture, which maintains a highly organized design, tuned to deliver the best performance within the Unity Game Engine (akin to Unity's new ECS/DOTS path). This architecture aims to break the chains to old monolithic approaches for building solutions in Unity.
+
+### These are the main differences in the XRTK
+
+| Difference | Comment |
+|---|---|
+| UPM (Unity Package Management), submodules and Unity asset delivery | XRTK offers a more Unity focused development channel than that of the enterprise-focused MRTK, utilizing Unity's built-in packaging system.  Additionally, we have streamlined the developer pattern aligned with how community devs interact.
```
Comment (StephenHodgson) [inline articles/A02-ChoosingXRTKvsMRTK.md]: This table isn't rendering properly

File: articles/A02-ChoosingXRTKvsMRTK.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### In layman's terms
+
+The MRTK focuses on delivery to new and pre-existing partner solutions and as such, can be limited in the paths it can take.  XRTK is more open and can easily adapt and educate the community to build almost any solution without compromise.
+
+## Supported Platforms
+
+
+Platform | Description | XRTK Support | MRTK Support
+---|---|---|---|---|
```
Comment (StephenHodgson) [inline articles/A02-ChoosingXRTKvsMRTK.md]: image

**Comment by SimonDarksideJ** [comment]
Fixed table

**Comment by StephenHodgson** [comment]
Can we do bullet points for the table items with multiple values?
image

**Comment by StephenHodgson** [comment]
Also what does tbc mean?
